libtkldet/modman.py

```diff
@@ -19,11 +19,15 @@
 
 import importlib.machinery
 import importlib.util
-from os.path import join, dirname, abspath, splitext, isfile
+from os.path import join, dirname, abspath, splitext, isfile, exists
 from os import listdir
 import sys
 
 from . import colors as co
+from .error import TKLDevDetectiveError
+
+# priortise local tkldet_modules path, fallback to OS path
+MOD_PATH = [ abspath(__file__), '/usr/share/tkldev-detective']
```

Comment:
pretty sure it should be `dirname(dirname(abspath(__file__)))` not just `abspath(__file__)`

Reply:
Thanks @OnGle, good catch!

I didn't actually test that, so you're probably right. I'll double check right now and report back.

Reply:
Bug confirmed, and fixed!